.gitignore

```diff
@@ -1 +1,2 @@
 venv
+__pycache__
```

Comment:
We could probably put a "proper" Python `.gitignore` in here: https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
@Shillaker should we copy and paste this here?

Reply:
Yeah just copy-paste. Maybe remove some of the bits we know we'll never need but it's not a big deal